content/updates/quality/1-The-quality-score.md

```diff
@@ -0,0 +1,32 @@
+# Introducing our Quality score!
+
+## A new way of scoring...
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/understand-data-quality.html#about-data-quality-insights"
+:::
+Measuring your product quality is now easier than ever thanks to the **quality score**, our new simplified way to score your product. It replaces the enrichment and consistency grades for better clarity.
+
+![The new quality score](../img/quality-score.png)
+
+You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to use it as a filter to focus your attention on the products with low-quality scoring.
+
+![The new quality score in the product edit form](../img/quality-score-in-pef.png)
+
+## ...All your products...
+::: meta-data type="New" ee-only="false"
+:::
+You'll also notice that this score and the Data Quality Insights tab are now available for both your variant products and product models! You will get specific quality scores for all your variant products so you can quickly identify the list of attributes you need to improve for a specific variant product. We also deliver a quality score at the product model level so you can see at a glance on which level the attributes need to be improved (at the variant product or at the product model level).
+
+![DQI on variant products](../img/dqi-for-a-variant-product.png)
+
+When you have improvement recommendations, you can simply click on your attribute to go directly to it and improve it under the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.
+
+## ...On selected attributes!
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-data-quality.html#data-quality-insights-activation-and-deactivation"
+:::
+
+You can now select the product attributes to take into account when calculating your product quality score. You can toggle this option at the attribute group level under the attribute group settings page.
```

Comment:
```suggestion
You can now select the product attributes you want to take into account when calculating your product quality score. You can enable this option at the attribute group level under the attribute group settings page.
```

Reply:
What do you mean by 'under the attribute group' -> shouldn't it be ON the (...) page?

Reply:
On each attribute group page, you have a toggle button to choose whether you want it to be part of the quality score calculation.

Reply:
ok, so I guess it's on, under would mean it's physically under another component

Reply:
```suggestion
You can now select the product attributes you want to take into account when calculating your product quality score. You can enable this option at the attribute group level on the attribute group settings page.
```